extra-dev/packages/coq-mathcomp-real-closed/coq-mathcomp-real-closed.dev/opam

```diff
@@ -1,19 +1,21 @@
 opam-version: "2.0"
 name: "coq-mathcomp-real-closed"
-version: "dev"
 maintainer: "Mathematical Components <mathcomp-dev@sympa.inria.fr>"
 
-homepage: "https://math-comp.github.io/math-comp/"
-bug-reports: "Mathematical Components <mathcomp-dev@sympa.inria.fr>"
+homepage: "https://github.com/math-comp/real-closed"
+bug-reports: "https://github.com/math-comp/real-closed/issues"
 dev-repo: "git+https://github.com/math-comp/real-closed.git"
 license: "CeCILL-B"
 
 build: [ make "-j" "%{jobs}%" ]
 install: [ make "install" ]
-remove: [ "sh" "-c" "rm -rf '%{lib}%/coq/user-contrib/mathcomp/real_closed'" ]
-depends: [ "coq-mathcomp-field" { (= "1.7.0") } ]
+depends: [
+  "coq" { (>= "8.7" & < "8.10~") }
+  "coq-mathcomp-field" { (>= "1.8.0" & < "1.9.0~") }
+  "coq-mathcomp-bigenough"   {(>= "1.0.0" & < "1.1.0~")}
+]
 
-tags: [ "keyword:real closed field" "keyword:small scale reflection" "keyword:mathematical components" ]
+tags: [ "keyword:real closed field" "keyword:small scale reflection" "keyword:mathematical components" "date:2019-05-23" "logpath:mathcomp"]
```

Comment:
Note that there is no need to have `date` metadata for `extra-dev` packages (since a user will get new releases whenever the source repo is updated).